src/app/auth/auth.component.ts

```diff
@@ -159,6 +159,7 @@ export class AuthComponent implements OnInit, OnDestroy {
     this.authService.changeAuthState(AuthState.AwaitingAuthentication);
     this.phaseService.setPhaseOwners(this.currentSessionOrg, username);
     this.userService.createUserModel(username).pipe(
+      flatMap(() => this.phaseService.storeSessionData()),
       flatMap(() => this.phaseService.sessionSetup()),
```

Comment:
There seems to be a bit of duplication between `phaseService.storeSessionData` and `phaseService.sessionSetup`.
In this implementation, `phaseService.storeSessionData` will fetch the settings file and save its contents to local storage. `phaseService.sessionSetup` will then read the settings data from the cache.

Both methods will validate the contents of the settings' file.
I think we can actually remove `phaseService.storeSessionData` entirely

Reply:
:+1: sounds good to me